pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -5307,6 +5307,12 @@
         "uap10.0.16299": "4.0.1.0"
       }
     },
+    "System.Utf8String": {
```

Comment:
When we have done similar experimental package for hardware intrinsics, we added `.Experimental` suffix to the name to make it clear that it is experimental package. Do the same here?